_woc/WoCabulary.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "WoCabulary - My Woc Blog"
```

Comment:
Innovative name there! Great.

Reply:
If you need help with the blog :
https://github.com/khanfarhan10/khanfarhan10.github.io/pull/2/files
https://github.com/khanfarhan10/khanfarhan10.github.io/pull/3/files
https://github.com/khanfarhan10/khanfarhan10.github.io/pull/1/files